DESCRIPTION

```diff
@@ -24,7 +36,11 @@ Enhances:
     limma,
     edgeR,
     Seurat (>= 3.0),
-    scater
+    scater,
+    scran, 
+    scuttle,
+    SingleCellExperiment,
+    ReactomeGSA
```

Comment:
Something I just learned: You can use [usethis::use_tidy_description()](https://usethis.r-lib.org/reference/tidyverse.html) to automatically format the DESCRIPTION file